lib/modules/dosomething/dosomething_reportback/includes/dosomething_reportback_file.inc

```diff
@@ -58,17 +58,16 @@ class ReportbackFileEntity extends Entity {
       // Store that instead.
       $this->review_source = $values['source'];
     }
-    if ($this->status === 'flagged') {
```

Comment:
I didn't know that was a requirement.
is there a different place I should put this code?